### PR TITLE
First assignment final.

### PR DESCRIPTION
**Pull request by Alex-A14**
When I first inserted a picture I wasn't happy with the size. However, I found a code format online which allowed me to change the height and width of the picture. 
I inserted a picture from an online link.  I wasn't sure how to insert a picture saved on my computer. Do I have to show the exact path to the picture?
I had some trouble with sub-lists but was able to get them working when I put spaces between each line.
I was easily able to attach my link to text rather than just pasting the link.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,32 @@
...
-Repository for first CFSS homework assignment
+# ~~hw01~~
+~~Repository for first CFSS homework assignment~~
+
+# Alex Albury Bio
+Hello all (or should I say 'Hello World'?),
+
+I'm an international student from The Bahamas(see islands in yellow below).
+
+<img src="http://i.infopls.com/images/mbahama.gif" width="350" height="350" />
```
Comment (gmvelez) [inline README.md:9]: Good job changing the shape of the image by finding out how to add the extra code!

File: README.md [hunk 2]
```diff
@@ -1,2 +1,32 @@
...
+
+My interests cover:
+
+- Cognitive Neuroscience
+
+- Language
+
+    - Bilingualism
+  
+    - Linguistic Relativity
```
Comment (gmvelez) [inline README.md:21]: Good job with ordered lists!

File: README.md [hunk 3]
```diff
@@ -1,2 +1,32 @@
...
+
+Basically, I'm interested in how language affects how we experience the world and I think one of the best ways to understand experience is by studying the brain. 
+
+I get to explore these interests as an RA in the [Expereince and Cognition Lab](http://www.casasanto.com/).
+
```
Comment (gmvelez) [inline README.md:26]: I like this as a detail about yourself and a good way to incorporate a link

**Comment by gmvelez** [comment]
Evaluation HW-01 for Alex-A14

| **Coding style**  N/A
| **Coding strategy**    N/A
| **Presentation**      +
| **Achievement, creativity** +
| **Ease of access**        +

Remarks:
This was a solid biography using a number of different elements that were incorporated well into the file.  I especially thought the resizing of the image was a creative touch, and the use of a link to the lab where Alex works.

**Comment by jmausolf** [comment]
## Evaluation HW-01 for Alex Albury

| Topic | Excellent | Satisfactory | Needs Work |
| --- | --- | --- | --- |
| **Coding style** |        x  |     |             |
| **Coding strategy** |     x   |        |        |
| **Presentation: graphs** |      |     |       |
| **Presentation: tables** |       |       |     |
| **Achievement, creativity** |x      |      |         |
| **Ease of access** | x |                 |             |

Remarks:
- Please put your name *Alex Albury* in the commit message. This helps us easily know whose homework is whose as github usernames are often ambiguous.
- For the header of the assignment, you probably could have deleted the "hw01" header and subsequent lines versus using strikethrough. Good use of markdown, but not really the most polished look to begin your bio.
- To insert a picture on your computer, you can also commit that photo, say `my_photo.jpg` and then add the line `My bio pic` to your markdown file.
- Nice "hello world" joke and creative inclusion of a map instead of a picture.
- Otherwise, you effectively used four of the five style elements, headers, list, photo, and link. Good job.
- In general try to make more frequent commits versus waiting until the final work is done.
- Good job overall.

**Comment by nhonors** [comment]
Evaluation HW-01 for Alex
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**      |   x|
| **Coding strategy**   |   x|
| **Presentation**        |   x|
| **Achievement, creativity**|x|
| **Ease of access**      |  x|

Remarks:

It looks like you have a strong handle on this stuff. Want to help me? I'm lost! 
Also, great info in your bio. Interesting stuff.
Your coding looks complex and efficient, as far as I can tell. Great job!

**Pull request closed without merging**